Lib/importlib/_bootstrap_external.py

```diff
@@ -358,7 +359,7 @@ def _write_atomic(path, data, mode=0o666):
 # Whenever MAGIC_NUMBER is changed, the ranges in the magic_values array
 # in PC/launcher.c must also be updated.
 
-MAGIC_NUMBER = (3436).to_bytes(2, 'little') + b'\r\n'
+MAGIC_NUMBER = (3437).to_bytes(2, 'little') + b'\r\n'
```

Comment:
Maybe we won't have to bump this for a second time in a week? Bytecode generated with this version can be executed with the previous version and vice versa.

Reply:
:+1: